data_safe_haven/commands/config.py

```diff
@@ -22,11 +22,12 @@ def template(
 ) -> None:
     """Write a template Data Safe Haven configuration."""
     config = Config.template()
+    config_yaml = config.to_yaml(warnings=False)
```

Comment:
Possibly worth commenting on *why* we want to disable warnings here?

Reply:
Fixed in 8b61e7b